### PR TITLE
ciscoPacketTracer{7,8}: disown

### PR DESCRIPTION
**Pull request by lucasew**
## Description of changes
When I introduced these packages I needed them for Networking class but now I don't use them anymore.

Actually I wanted to drop them all out but because of activity in #291445 I think it would be better to let someone else maintain as there is some interest in these packages.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by gepbird** [comment]
@lucasew can you add me as a maintainer?

I'll be using this for a few years on uni.

**Comment by lucasew** [comment]
> @lucasew can you add me as a maintainer?
> 
> I'll be using this for a few years on uni.

You can add yourself the next time you open a PR for it. This maintainer thing is mostly for someone to tag you if the package has some issue.

**Comment by gepbird** [comment]
Fine, I'll add myself in a seperate PR, I just thought its easier to do it in this.

**Review by gepbird — APPROVED** [review]
Thanks for creating the package, see ya

**Review by thiagokokada — APPROVED** [review]

File: pkgs/by-name/ci/ciscoPacketTracer7/package.nix [hunk 1]
```diff
@@ -84,7 +84,7 @@ in stdenv.mkDerivation {
     homepage = "https://www.netacad.com/courses/packet-tracer";
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     license = licenses.unfree;
-    maintainers = with maintainers; [ lucasew ];
+    maintainers = [];
```
Comment (thiagokokada) [inline pkgs/by-name/ci/ciscoPacketTracer7/package.nix:87]: In general, we keep in the `with maintainers;` even if this is empty:
```suggestion
    maintainers = with maintainers; [ ];
```

File: pkgs/by-name/ci/ciscoPacketTracer8/package.nix [hunk 2]
```diff
@@ -106,7 +106,7 @@ stdenvNoCC.mkDerivation (args: {
     homepage = "https://www.netacad.com/courses/packet-tracer";
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     license = licenses.unfree;
-    maintainers = with maintainers; [ lucasew ];
+    maintainers = [];
```
Comment (thiagokokada) [inline pkgs/by-name/ci/ciscoPacketTracer8/package.nix:109]:
```suggestion
    maintainers = with maintainers; [ ];
```

**Review by thiagokokada — APPROVED** [review]

**Pull request merged**